app/helpers/offender_helper.rb

```diff
@@ -88,7 +92,7 @@ def mappa1_handover_date(offender)
     ].compact.min
   end
 
-  def parole_handover_date(offender)
+  def parole_responsibility_date(offender)
     offender.parole_eligibility_date - 8.months
```

Comment:
Do we care that this may be nil? Unfortunately we can't make any safe assumptions about anything that came from sentence details :(